pkg/privilege/privileges/cache.go

```diff
@@ -1913,7 +1913,7 @@ func (p *MySQLPrivilege) showGrants(ctx sessionctx.Context, user, host string, r
 }
 
 type columnStr = string
-type columnStrs = []columnStr
+type columnStrs = map[columnStr]struct{}
```

Comment:
Does the column privilege feature also exists in master branch?

Reply:
No, we have not implemented the checking of column-level privilege. But you can grant column privilege and show grants with column privilege.